test_xdb/models/under_test/swap_schema_test.sql

```diff
@@ -0,0 +1,38 @@
+{{config({
+    "tags":["exclude_bigquery", "exclude_bigquery_tests"],
+    "pre-hook": [{"sql": "CREATE SCHEMA schema_one;
+                          CREATE TABLE schema_one.table_1 AS select 1 AS column_1;
+                          CREATE VIEW schema_one.view_1 AS select 1 AS column_1;
+                          CREATE SEQUENCE schema_one.sequence_1;
+                          CREATE SCHEMA schema_two;
+                          CREATE TABLE schema_two.table_2 AS select 2 AS column_2;
+                          CREATE VIEW schema_two.view_2 AS select 2 AS column_2;
+                          CREATE SEQUENCE schema_two.sequence_2;"
+                          }, 
+                "{{ xdb.swap_schema('schema_one', 'schema_two') }}"],
+    "post-hook": [{"sql": "DROP SCHEMA schema_one CASCADE;
+                           DROP SCHEMA schema_two CASCADE;"}]})
+}}
+
+WITH test_objects_metadata AS (
+    SELECT
+        LOWER(table_type) AS object_type
+        , table_schema AS schema_name
+        , table_name AS object_name
+    FROM information_schema.tables
+    WHERE table_schema IN ('schema_one','schema_two')
+    UNION
```

Comment:
can be union all since we are applying the max function

Reply:
I think it doesn't matter in this particular case, but I can add it.